app/src/main/python/gogdl/dl/managers/manager.py

```
@@ -0,0 +1,207 @@
+        # Of course assuming they will ever change their content system generation (I highly doubt they will)
+        if generation not in [1, 2]:
+            raise Exception("Unsupported depot version please report this")
+
+        self.logger.info(f"Depot version: {generation}")
+
+        if generation == 1:
+            self.download_manager = v1.Manager(self)
+        elif generation == 2:
+            self.download_manager = v2.Manager(self)
+
+    def calculate_download_size(self, arguments, unknown_arguments):
```

Ruff (0.12.2)

Unused method argument: `arguments`

(ARG002)